packages/jest-snapshot/src/utils.js

```diff
@@ -179,3 +187,58 @@ export const saveSnapshotFile = (
     writeSnapshotVersion() + '\n\n' + snapshots.join('\n\n') + '\n',
   );
 };
+
+export const saveInlineSnapshots = (
+  snapshots: InlineSnapshot[],
+  sourceFilePath: Path,
+) => {
+  const sourceFile = fs.readFileSync(sourceFilePath, 'utf8');
+
+  const config = prettier.resolveConfig.sync(sourceFilePath);
+  const newSourceFile = prettier.format(
+    sourceFile,
+    Object.assign({}, config, {
+      filepath: sourceFilePath,
+      parser: createParser(snapshots),
+    }),
+  );
+
+  if (newSourceFile !== sourceFile) {
+    fs.writeFileSync(sourceFilePath, newSourceFile);
+  }
+};
+
+const createParser = snapshots => (text, parsers) => {
+  const ast = parsers.babylon(text);
+  traverse(ast, {
+    CallExpression({node}) {
+      if (
+        node.callee.type !== 'MemberExpression' ||
+        node.callee.property.type !== 'Identifier'
+      ) {
+        return;
+      }
+      const matcher = node.callee.property;
+      for (const {snapshot, frame} of snapshots) {
+        if (
+          matcher.loc.start.line === frame.line &&
+          matcher.loc.start.column === frame.column - 1
+        ) {
+          if (
+            node.arguments[0] &&
```

Comment:
Note to self: It won't always be the first argument, it should always be the last one. Property matchers should probably go first.

```js
expect(x).toMatchInlineSnapshot(
  { r: expect.any(Number) },
  `
  Object {
    r: Any<Number>
  }
`);

Reply:
✅

Reply:
😍